src/Raven.Server/Web/Studio/Processors/BucketsHandlerProcessorForGetBuckets.cs

```diff
@@ -19,7 +19,7 @@ public BucketsHandlerProcessorForGetBuckets([NotNull] DatabaseRequestHandler req
         {
         }
         
-        protected override ValueTask<BucketsResults> GetBucketsResults(DocumentsOperationContext context, int fromBucket, int toBucket, int range, CancellationToken token)
+        protected override ValueTask<BucketsResults> GetBucketsResults(DocumentsOperationContext context, int fromBucket, int toBucket, int range, int? _, CancellationToken token)
```

Comment:
maybe better to get the shardNumber params only from the sharded version?

Reply:
Changed